src/models/category.js

```diff
@@ -7,6 +7,18 @@ const categorySchema = new Schema({
     type: String,
     unique: true,
     required: [true, FIELD_CANNOT_BE_EMPTY('name')]
+  },
+  appearance: {
+    icon: {
+      type: String,
+      required: [true, FIELD_CANNOT_BE_EMPTY('icon')],
+      default: 'https://www.svgrepo.com/show/532031/cloud-fog.svg'
+    },
+    color: {
+      type: String,
+      required: [true, FIELD_CANNOT_BE_EMPTY('color')],
+      default: '#66C42C'
```

Comment:
It might be better to use either the `default` value or the `required` validator, but not both

Reply:
I think I will ask Natalya about it, because in task description both required validator with error handling from consts and default values are mentioned.

Reply:
Let's leave default values only. Otherwise, the user will be forced to set those values and defaults will be never applied